### ISSUE TITLE
Build fails on RN v0.69.0-rc.1

### ISSUE THREAD

**Issue by foo-bar-dr** [ammarahm-ed/react-native-mmkv-storage#255]
**Describe the bug**
On upgrading to React Native version 0.69.0-rc.1, running `npx react-native run-android` command fails with the following error:

```
FAILURE: Build failed with an exception.

* Where:
Build file 'C:\Projects\rnplayground\node_modules\react-native-mmkv-storage\android\build.gradle' line: 120

* What went wrong:
A problem occurred evaluating project ':react-native-mmkv-storage'.
> Expected directory 'C:\Projects\rnplayground\node_modules\react-native-mmkv-storage\android\..\..\..\node_modules\react-native\android' to contain exactly one file, however, it contains more than one file.
```

**To Reproduce**
1. Upgrade to react native's latest version (release candidate 0.69.0-rc.1)
2. Run `npx react-native run-android`

**Expected behavior**
The build should succeed without any error

**Platform Information:**
 - OS: Android
 - React Native Version [e.g. 0.69.0-rc.1]
 - Library Version 0.7.5

**Comment by ammarahm-ed** [comment]
@foo-bar-dr Do you have the new arch enabled?

**Comment by foo-bar-dr** [comment]
Build fails irrespective of the new arch. I have tried enabling and disabling new arch but it fails on both while configuring the build.

**Comment by ammarahm-ed** [comment]
@foo-bar-dr It seems RN 0.69 has switched to seperate files for release & debug.

**Comment by ammarahm-ed** [comment]
Fixed in v0.7.6
